mapsforge-samples-awt/src/main/java/org/mapsforge/samples/awt/Samples.java

```diff
@@ -181,21 +186,23 @@ private static MapView createMapView() {
     }
 
     @SuppressWarnings("unused")
-    private static TileDownloadLayer createTileDownloadLayer(TileCache tileCache, IMapViewPosition mapViewPosition, TileSource tileSource) {
+    private static TileDownloadLayer createTileDownloadLayer(TileCache tileCache, MapView mapView, IMapViewPosition mapViewPosition, TileSource tileSource) {
         return new TileDownloadLayer(tileCache, mapViewPosition, tileSource, GRAPHIC_FACTORY) {
             @Override
             public boolean onTap(LatLong tapLatLong, Point layerXY, Point tapXY) {
-                System.out.println("Tap on: " + tapLatLong);
+                final byte currentZoomLevel = mapView.getModel().mapViewPosition.getZoomLevel();
+                System.out.println("Tap on: " + tapLatLong + ", zoom level: " + currentZoomLevel);
                 return true;
             }
         };
     }
 
-    private static TileRendererLayer createTileRendererLayer(TileCache tileCache, MapDataStore mapDataStore, IMapViewPosition mapViewPosition, HillsRenderConfig hillsRenderConfig) {
+    private static TileRendererLayer createTileRendererLayer(TileCache tileCache, MapDataStore mapDataStore, MapView mapView, IMapViewPosition mapViewPosition, HillsRenderConfig hillsRenderConfig) {
```

Comment:
Zoom level can be retrieved from already existing `mapViewPosition`.

Reply:
Btw, is it okay to have field names start with "m" (e.g. mZipFile) in our own code to avoid cluttering up with "this."?